pages/docs/reference/coding-conventions.md

```diff
@@ -726,6 +727,195 @@ foo {
 
 </div>
 
+### Trailing commas
+
+Use trailing commas at declaration site, including multi-line enumerations such as argument and parameter lists, `when` entries, and components of destructuring declarations.
+For call site trailing commas are optional.
```

Comment:
Isn't it optional at the declaration site as well?

Reply:
Maybe we should explain separately that it's _technically_ optional at both sites, but _as a convention_ we recommend (what we recommend :)).